packages/webapp/src/pages/_app.tsx

```diff
@@ -24,6 +24,7 @@ import { UalSoftkeyLoginModal } from "_app/eos/ual/softkey";
 import "tailwindcss/tailwind.css";
 import "_app/styles/nprogress.tailwind.css";
 import "_app/styles/add-to-calendar.chq.css";
+import "react-virtualized/styles.css"; // only needs to be imported once
```

Comment:
suggestion: remove redundant comment since we are assuming all of the above css are always being imported once